external/lein-repo/src/lein_repo/plugin.clj

```diff
@@ -4,7 +4,7 @@
             [clojure.set :as set]
             [clojure.pprint :as pprint]
             [clojure.java.shell :as shell]
-            leiningen.core.project)
+            [leiningen.core.project :as project]
```

Comment:
Yeesh! I forgot to try rebuilding it after I removed a bunch of debugging stuff. Thanks for pointing that out.